src/os/bluestore/BlueStore.h

```diff
@@ -2334,6 +2334,24 @@ class BlueStore : public ObjectStore,
   int _fsck(bool deep, bool repair);
 
   void set_cache_shards(unsigned num) override;
+  void dump_cache_stats(Formatter *f) override {
```

Comment:
@gregsfortytwo @chardan printing the cache stats is now offloaded to the object store. There is no more function `get_cache_object_count` that can be called from the OSD to get the cache stats. I think this makes more sense, since the cache stats mean different things for different object stores.